src/routers/controllers/stopPageAddOwnerControllerGet.ts

```diff
@@ -1,27 +1,32 @@
-import { Request, RequestHandler, Response } from "express";
+import { Request, Response } from "express";
 import * as constants from "../../lib/constants";
 import { getTranslationsForView } from "../../lib/utils/translationUtils";
 import { getExtraData, getLoggedUserAcspMembership } from "../../lib/utils/sessionUtils";
 import { AcspMembership } from "private-api-sdk-node/dist/services/acsp-manage-users/types";
 import { MemberForRemoval } from "types/membership";
 import logger from "../../lib/Logger";
 import { getRemoveMemberCheckDetailsFullUrl } from "../../lib/utils/urlUtils";
+import { UserRoleChangeData } from "types/utilTypes";
 
-export const stopPageAddOwnerControllerGet: RequestHandler = async (req: Request, res: Response) => {
+export const stopPageAddOwnerControllerGet = async (req: Request, res: Response): Promise<void> => {
     const loggedUserAcspMembership: AcspMembership = getLoggedUserAcspMembership(req.session);
     const userToRemove: MemberForRemoval | undefined = getExtraData(req.session, constants.DETAILS_OF_USER_TO_REMOVE);
+    const userToChangeRole: UserRoleChangeData | undefined = getExtraData(req.session, constants.USER_ROLE_CHANGE_DATA);
 
-    if (!userToRemove) {
-        logger.error(`${stopPageAddOwnerControllerGet.name}: DETAILS_OF_USER_TO_REMOVE not found in session`);
-        throw Error("DETAILS_OF_USER_TO_REMOVE not found in session");
+    if (!userToRemove && !userToChangeRole) {
+        logger.error(`${stopPageAddOwnerControllerGet.name}: neither DETAILS_OF_USER_TO_REMOVE nor DETAILS_OF_USER_TO_REMOVE found in session`);
```

Comment:
is this intentional? DETAILS_OF_USER_TO_REMOVE is repeated - DETAILS_OF_USER_TO_REMOVE nor DETAILS_OF_USER_TO_REMOVE

Reply:
Ah, this is an error. The second one should be USER_ROLE_CHANGE_DATA. Well spotted.